benchmarks/text_hybrid_scoreboard/sqlite_fts5_bench.py

```
@@ -0,0 +1,202 @@
+    parser.add_argument("--db", default="", help="SQLite DB path; defaults to a file beside --out")
+    parser.add_argument("--keep-db", action="store_true")
+    parser.add_argument("--tokenize", default="unicode61")
+    return parser.parse_args()
+
+
+def write_json(path: Path, payload: dict[str, Any]) -> None:
+    path.parent.mkdir(parents=True, exist_ok=True)
+    path.write_text(json.dumps(payload, indent=2, sort_keys=True) + "\n", encoding="utf-8")
+
+
+def unavailable(args: argparse.Namespace, reason: str) -> dict[str, Any]:
```

Ruff (0.15.15)

[warning] Unused function argument: `args`

(ARG001)